pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -329,12 +329,14 @@ public static class Broker {
 
     // Broker config indicating the maximum serialized response size across all servers for a query. This value is
     // equally divided across all servers processing the query.
+    // The value can be in human readable format (e.g. '150K', '150KB', '0.15MB') or in bytes (e.g. '150000').
     public static final String CONFIG_OF_MAX_QUERY_RESPONSE_SIZE_BYTES = "pinot.broker.max.query.response.size.bytes";
```

Comment:
Ideally we want to remove `.bytes` suffix here. Since this feature is not released yet (added after `1.0`), I guess we can change it? cc @vvivekiyer